### PR TITLE
Make the rows/columns/values text in pivot table settings stand out more

### PR DESCRIPTION
**Pull request by mazameli**
See screenshot. This probably requires updating strings in Poeditor 'cause I've modified the string templates here, but I'm already going to have to quickly update strings because of https://github.com/metabase/metabase/pull/14413, so I figured I'd go ahead and make this change as well.

Screen Shot 2021-01-15 at 9 36 10 AM

### CODE REVIEW

File: frontend/src/metabase/visualizations/visualizations/PivotTable.jsx [hunk 1]
```diff
@@ -19,17 +19,23 @@ const partitions = [
   {
     name: "rows",
     columnFilter: isDimension,
-    title: t`Fields to use for the table rows`,
+    title: jt`Fields to use for the table ${(
+      <span className="text-dark text-heavy">rows</span>
```
Comment (paulrosenzweig) [inline frontend/src/metabase/visualizations/visualizations/PivotTable.jsx:23]: I think you have to mark "rows" for translation too.

**Review by paulrosenzweig — APPROVED** [review]
Is it ok to update the strings with so little lead time for translators? I guess there's only a few to translate.

File: frontend/src/metabase/visualizations/visualizations/PivotTable.jsx (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/visualizations/visualizations/PivotTable.jsx]: Ah, you're right, I got the syntax wrong. Pushed up a fix.

**Comment by mazameli** [comment]
Yeah, I'm going to auto translate them and not ask translators to do it, since we did that with many strings in many translations to get them from e.g. 99% to 100%.

**Pull request merged**